packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -89,8 +89,12 @@ const getOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedPa
 
   let gridParticipants = newGridParticipants;
   let overflowGalleryParticipants = newOverflowGalleryParticipants;
```

Comment:
We immediately overwrite `overflowGalleryParticipants` in the if-else. It reassigns to either `overflowGalleryCallingParticipants` or `overflowGalleryParticipants.concat(callingParticipants);`

so it looks like the value of newOverflowGalleryParticipants is being thrown out unused?

Reply:
Oh wait in the else it concats with gridCallingParticipants